Documentation/ceph-upgrade.md

```diff
@@ -259,10 +259,31 @@ At this point, your Rook operator should be running version `rook/ceph:v1.0.0`
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
+### 6. Update the Mon Ports
+
+The port on which the mons are listening has changed in the 1.0 release. Previously, the mons were listening on
+port 6790 by default. In 1.0 now they are expected to be listening on port 6789. To ensure an uninterrupted
+upgrade to 1.0, the operator configures the existing mons to continue listening on port 6790.
+
+**Before upgrading to Ceph Nautilus,** it is recommended that the mons be re-configured to listen on
```

Comment:
I think beginning with this statement in bold is likely to cause users to infer that these instructions don't work once the cluster has been updated to Nautilus. Perhaps better ...
```
It is recommended that the mons be re-configured to listen on the new default port **before upgrading to Ceph Nautilus**; however the steps outlined below will still work after an upgrade to Nautilus.
```